stix2elevator/convert_cybox.py

```diff
@@ -938,18 +1022,21 @@ def convert_network_socket(socket, obj1x_id):
             warn("%s is not a member of the %s enumeration", 627, socket.address_family, "address family")
     if socket.type_:
         socket_extension["socket_type"] = socket.type_
-    if socket.domain and get_option_value("spec_version") == "2.0":
-        socket_extension["protocol_family"] = socket.domain
+    if socket.domain:
+        if get_option_value("spec_version") == "2.0":
+            socket_extension["protocol_family"] = socket.domain
+        else:
+            handle_missing_string_property(socket_extension, "protocol_family", socket.domain, s_sco=True)
```

Comment:
I think there is a minor typo here. Should it be `is_sco`?

```suggestion
            handle_missing_string_property(socket_extension, "protocol_family", socket.domain, is_sco=True)
```